packages/MSBot/src/msbot.ts

```diff
@@ -65,22 +66,27 @@ program
 program
     .command('update <service>', 'update a service record (Luis/Qna/Azure/...) used by the bot');
 
-
-const args: program.Command = program.parse(process.argv);
-
-// args should be undefined is subcommand is executed
-if (args) {
-    const unknownArgs: string[] = process.argv.slice(2);
-    console.error(chalk.default.redBright(`Unknown arguments: ${unknownArgs.join(' ')}`));
-    program.outputHelp((str: string) => {
-        console.error(str);
-
-        return '';
-    });
-    process.exit(1);
-}
+(async () => {
+    const latest: string = await latestVersion(pkg.name);
+    if (semver.gt(latest, pkg.version)) {
+        process.stderr.write(chalk.default.yellowBright(`\nNew version ${latest} is available to install.\n\n`))
```

Comment:
Consider replacing this with these lines (reviewed text with design). 

        process.stderr.write(chalk.default.white(`\n     Update available`));
        process.stderr.write(chalk.default.gray(` ${pjson.version}`));
        process.stderr.write(chalk.default.white(` -> `));
        process.stderr.write(chalk.default.greenBright(`${latest}\n`));
        process.stderr.write(chalk.default.white(`       Run `));
        process.stderr.write(chalk.default.blueBright(`npm i -g ${pjson.name}`));
        process.stderr.write(chalk.default.white(` to update.\n\n`));